pysatMissions/instruments/missions_sgp4.py

```diff
@@ -138,73 +148,84 @@ def load(fnames, tag=None, inst_id=None, obs_long=0., obs_lat=0., obs_alt=0.,
     if num_samples is None:
         num_samples = 100
 
-    # Create satellite from TLEs and assuming a gravity model;
-    # according to module webpage, wgs72 is common
-    satellite = twoline2rv(line1, line2, wgs72)
-
     # Extract list of times from filenames and inst_id
     times, index, dates = ps_meth.generate_times(fnames, num_samples,
                                                  freq=cadence)
-
-    # Create list to hold satellite position, velocity
-    position = []
-    velocity = []
-    for timestep in index:
-        # orbit propagator - computes x,y,z position and velocity
-        pos, vel = satellite.propagate(timestep.year, timestep.month,
-                                       timestep.day, timestep.hour,
-                                       timestep.minute, timestep.second)
-        position.extend(pos)
-        velocity.extend(vel)
+    # Calculate epoch for orbital propagator
+    epoch = (dates[0] - dt.datetime(1949, 12, 31)).days
+    jd, _ = jday(dates[0].year, dates[0].month, dates[0].day, 0, 0, 0)
+
+    if inclination:
```

Comment:
```suggestion
    if inclination is not None:
```
I prefer to be a bit more strict with checking against None.